stacks-common/src/types/mod.rs

```diff
@@ -92,6 +93,7 @@ impl StacksEpochId {
             | StacksEpochId::Epoch22
             | StacksEpochId::Epoch23 => false,
             StacksEpochId::Epoch24 => true,
+            StacksEpochId::Epoch30 => true, //TODO: Is this correct, or should it only be the latest epoch which is true?
```

Comment:
That's correct, 2.4 and 3.0 should both value sanitize.

Reply:
Okay, comment removed